gemma-core/src/main/java/ubic/gemma/core/analysis/expression/diff/GeneDifferentialExpressionServiceImpl.java

```diff
@@ -55,36 +55,11 @@ public GeneDifferentialExpressionServiceImpl(
 
     @Override
     public ExperimentalFactorValueObject configExperimentalFactorValueObject( ExperimentalFactor ef ) {
-        ExperimentalFactorValueObject efvo = new ExperimentalFactorValueObject( ef.getId() );
-        efvo.setName( ef.getName() );
-        efvo.setDescription( ef.getDescription() );
-        Characteristic category = ef.getCategory();
-        if ( category != null ) {
-            efvo.setCategory( category.getCategory() );
-            efvo.setCategoryUri( category.getCategoryUri() );
-        }
-        Collection<FactorValue> fvs = ef.getFactorValues();
-        StringBuilder factorValuesAsString = new StringBuilder( StringUtils.EMPTY );
-
-        for ( FactorValue fv : fvs ) {
-            String fvName = fv.toString();
-            if ( StringUtils.isNotBlank( fvName ) ) {
-                factorValuesAsString.append( fvName ).append( GeneDifferentialExpressionServiceImpl.FV_SEP );
-            }
-        }
-
-        /* clean up the start and end of the string */
-        factorValuesAsString = new StringBuilder(
-                StringUtils.remove( factorValuesAsString.toString(), ef.getName() + ":" ) );
-        factorValuesAsString = new StringBuilder( StringUtils
-                .removeEnd( factorValuesAsString.toString(), GeneDifferentialExpressionServiceImpl.FV_SEP ) );
-
```

Comment:
This needs to be double-checked with the `ExperimentalFactorValueObject` constructor.